config/application.rb

```diff
@@ -39,13 +39,13 @@ class Application < Rails::Application
 
     config.middleware.insert_after Rails::Rack::Logger, Rack::Cors do
       allow do
-        origins /.*\.prx\.(?:org|dev)$/
+        origins /.*\.prx\.(?:org|dev|tech)$/
         resource '/api/*', methods: [:get, :put, :post, :options], headers: :any
       end
 
       allow do
         origins '*'
-        resource '/api/*', methods: [:get]
+        resource '/api/*', methods: [:get], headers: :any
```

Comment:
So except for unauthorized GET requests, we're keeping the CMS api internal-only for the time being?  That's fine - just want to be clear.